tests/shear_heating_with_melt.prm

```diff
@@ -187,12 +200,12 @@ subsection Melt settings
 end
 
 subsection Heating model
-  set List of model names                     = adiabatic heating, latent heat, shear heating, adiabatic heating of melt, shear heating with melt
+  set List of model names                     = adiabatic heating, latent heat melt, shear heating, adiabatic heating of melt, shear heating with melt
 end
 
 subsection Postprocess
 
-  set List of postprocessors = composition statistics,velocity statistics, mass flux statistics, heating statistics, temperature statistics
```

Comment:
the visualization was likely added by accident?

Reply:
I can remove it again, but if anyone needs to look at this test again in the future because something has changed, they will need the visualization output again, because there's no other way to really see what the test does and if its behaviour is still correct. So unless you feel strongly about removing it, I think it would be kind of useful to just leave it switched on.